### DISCUSSION TITLE
Macros

### DISCUSSION THREAD

**Discussion by infamous-panda** [terjeio/ioSender#146] [General]
Created a ton of macros in the previous beta version. Where are these stored? Is there a file somewhere I can copy to the current version.

**Comment by terjeio** [comment]
Settings and macros are stored Iin the _App.config_ file, copy it to the new version or overwrite the old with the new when unpacking. _App.config_ is created on first startup if not found.

**Reply by infamous-panda** [reply]
Thanks. Probably better if it were a separate file but certainly not a big deal.
